index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Selcuk Guler's Personal Portfolio website</title>
+  </head>
+  <body>
+    <main>
+      <div id="topbar">
+        <div id="appbar">
+          <div id="timer-cont">
+            <div id="timer-text">9:41</div>
+          </div>
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <nav id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </nav>
+        </div>
```

Comment:
- [ ] Fantastic work on implementing the requested changes from the previous reviewer! However, there is **one remaining change that has not been addressed yet**. Could you please consider using a `header` tag instead of a `div`? Utilizing a header tag aligns with **HTML best practices**. You can find more information about it in this [link](https://www.w3schools.com/tags/tag_header.asp).

  ![image](https://github.com/s2jack/create-portfolio-website1/assets/107952724/e454a3b7-059c-4cbd-a52d-ccf3a3ea706d)